docs/includes/eloquent-models/relationships/RelationshipController.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Models\Orbit;
+use App\Models\Planet;
+use Illuminate\Http\Request;
+
+class RelationshipController extends Controller
```

Comment:
I think you don't need to extend the `Controller` class (that doesn't exist in this namespace).